tests/test_trigger_bisect_jobs.py

```diff
@@ -197,6 +200,13 @@ def test_problems():
     openqa.main(args)
     openqa.openqa_clone.assert_not_called()
 
+    args.url = "http://openqa.opensuse.org/tests/101"
+    try:
+        openqa.main(args)
+    except FileNotFoundError as e:
+        if str(e) == "[Errno 2] No such file or directory: 'tests/data/python-requests/openqa.opensuse.org/tests/101/investigation_ajax'":
+            raise Exception('This test should have stopped due to passed job result before hitting this issue')
```

Comment:
If I encountered this error message I would likely wonder what "this issue" is referring to.